packages/z3-solver/meta.yaml

```diff
@@ -0,0 +1,14 @@
+package:
+  name: z3_solver
```

Comment:
I believe ``pyodide-build`` assumes that the directory name matches the package name and that's why the build package test fails [here](https://app.circleci.com/pipelines/github/pyodide/pyodide/4567/workflows/a43b1b5a-1ea3-48a3-8f5d-48b2eea27a82/jobs/52486?invite=true#step-105-56). So this should be ``z3-solver`` or the directory name should be ``z3_solver``.

I think the other test failure is indeed unrelated.

Reply:
> I believe `pyodide-build` assumes that the directory name matches the package name and that's why the build package test fails [here](https://app.circleci.com/pipelines/github/pyodide/pyodide/4567/workflows/a43b1b5a-1ea3-48a3-8f5d-48b2eea27a82/jobs/52486?invite=true#step-105-56). So this should be `z3-solver` or the directory name should be `z3_solver`.

Yes, that's right.